dt: 0.2          #Hz

massMatrix:
  m: 368         #kg
  xg: 0
  Xudot: 0
  Yvdot: 0
  Yrdot: 0
  Iz: 512
  Nrdot: 0

dampingMatrix:
  # Xu: 51.3
  Xu: 400
  Yv: 40.0
  Yr: 0
  Nv: 0
  Nr: 400.0

processNoiseCovariance:
  [2, 0, 0, 0, 0, 0, 0, 0,
  0, 2, 0, 0, 0, 0, 0, 0,
  0, 0, .5236, 0, 0, 0, 0, 0,
  0, 0, 0, 4, 0, 0, 0, 0,
  0, 0, 0, 0, 4, 0, 0, 0,
  0, 0, 0, 0, 0, .5236, 0, 0,
  0, 0, 0, 0, 0, 0, 3, 0,
  0, 0, 0, 0, 0, 0, 0, 3]

measurementNoiseCovariance:
  [.1, 0, 0, 0, 0, 0, 0, 0,
  0, .1, 0, 0, 0, 0, 0, 0,
  0, 0, .1, 0, 0, 0, 0, 0,
  0, 0, 0, .1, 0, 0, 0, 0,
  0, 0, 0, 0, .1745, 0, 0, 0,
  0, 0, 0, 0, 0, .2, 0, 0,
  0, 0, 0, 0, 0, 0, .1, 0,
  0, 0, 0, 0, 0, 0, 0, .1]

  # [.1, 0, 0, 0, 0, 0,
  # 0, .1, 0, 0, 0, 0,
  # 0, 0, .1745, 0, 0, 0,
  # 0, 0, 0, .2, 0, 0,
  # 0, 0, 0, 0, .1, 0,
  # 0, 0, 0, 0, 0, .1,]
